### ISSUE TITLE
Synchronize Spawn version with Buildings Library v9.1.0

### ISSUE THREAD

**Issue by dhblum** [ibpsa/project1-boptest#718]
Currently, the version of Spawn correlates to an old version of the Modelica Buildings Library.  This issue is to update the version to that used in the most recent version of the Modelica Buildings Library, v11.0.0.

The relevant code is at https://github.com/ibpsa/project1-boptest/blob/master/service/worker/Dockerfile#L13.

The version of Spawn for MBL v11.0.0 should be ``0.4.3-7048a72798``, as defined here: https://simulationresearch.lbl.gov/modelica/releases/v11.0.0/help/Buildings_ThermalZones_EnergyPlus_9_6_0_UsersGuide.html#Buildings.ThermalZones.EnergyPlus_9_6_0.UsersGuide.Installation.

This issue will also use the "light" version of Spawn so as to speed up download time.

**Comment by dhblum** [comment]
Before merging/completing this, should coordinate with issue #218, which is our first public test case that uses Spawn and I believe is tied to the Spawn version with Modelica Buildings Library v9.1.0. @terrancelu92 could confirm.

**Comment by terrancelu92** [comment]
@dhblum, yes it is tied with the Spawn version with Modelica Buildings Library v9.1.0.
I will check out and confirm if the test case is compatible to the light version of SPAWN_VERSION=0.3.0-8d93151657

**Comment by dhblum** [comment]
Thanks for confirming @terrancelu92.  You should be able to run your model with the light version.  But compiling with ``0.3.0-8d93151657`` probably won't work to simulate with ``0.4.3-7048a72798``.  I suggest we finalize the test case release before updating it to Buildings 11.

**Comment by dhblum** [comment]
FYI @kbenne

**Comment by dhblum** [comment]
The spawn version for Buildings v9.1, used in https://github.com/ibpsa/project1-boptest/issues/218, is ``0.3.0-0fa49be497`` (ref [here](https://simulationresearch.lbl.gov/modelica/releases/v9.1.0/help/Buildings_ThermalZones_EnergyPlus_9_6_0_UsersGuide.html#Buildings.ThermalZones.EnergyPlus_9_6_0.UsersGuide)).  Thus, I suggest changing https://github.com/ibpsa/project1-boptest/blob/master/service/worker/Dockerfile#L13 to ``ENV SPAWN_VERSION=light-0.3.0-0fa49be497``.

**Comment by dhblum** [comment]
Note that changing to use Spawn light reduces worker docker image to ~3.17 GB, reference to https://github.com/ibpsa/project1-boptest/issues/256.
